Perlang.Common/Expr.cs

```diff
@@ -81,7 +81,7 @@ public override string ToString()
             }
         }
 
-        public class Binary : Expr
+        public class Binary : Expr, ITokenAwareExpr
```

Comment:
(We should really strive for making _all_ expressions be token-aware, so we can produce proper error messages. Non-critical for now so will ignore it for the time being.)